### ISSUE TITLE
Connection Reject by server

### ISSUE THREAD

**Issue by NatS0e** [suriyun-mmorpg/UnityMultiplayerARPG_MMO.NET#2]
Hello, I'm having this issue right now when I used this .NET as a server. My Unity kit is the latest version.

![hyqnqShJ9W](https://github.com/user-attachments/assets/45998690-6673-44fa-84df-e5603d48c780)

**Comment by insthync** [comment]
Hello, try spawn map server with non batch mode, I want to see map-servers log messages (set "notSpawnInBatchMode": true in serverConfig.json)

**Comment by NatS0e** [comment]
![image](https://github.com/user-attachments/assets/8a2b4e97-7937-4693-b80f-43a3329e7d02)

**Comment by NatS0e** [comment]
Did a PC restart and got this.
![image](https://github.com/user-attachments/assets/3cb26005-e7ef-4d33-be80-04388af6c73c)

**Comment by insthync** [comment]
Yep, it should work

**Comment by NatS0e** [comment]
Yes it worked, but did you know If I try to clean the Solution or update the nuget packages , I get tons of errors.

**Comment by insthync** [comment]
Want to see an errors

**Comment by NatS0e** [comment]
Yes sir. 

![devenv_ajg4aLPnnb](https://github.com/user-attachments/assets/cede0028-7dee-456a-8699-0b0e6029482a)

**Comment by NatS0e** [comment]
I found this: https://github.com/Cysharp/ZLogger/blob/master/src/ZLogger/ZLoggerOptions.cs

**Comment by insthync** [comment]
Don't update it, if you don't know how to make changes to codes, I have to use the old version to make it not affects other customer projects
